### DISCUSSION TITLE
更新完2.0，真的难用，和刚做出来一样。

### DISCUSSION THREAD

**Discussion by MuFengss** [clash-verge-rev/clash-verge-rev#2351] [General]
每次调好的界面大小，下次再打开就恢复默认大小，修复了一下结果是在原有的大小上加大了一点，重启还是会重置大小，你都给我气笑了，tun模式也不生效，折腾半天还以为节点有问题，结果换v2就没事，你在干什么？看来是1.7.7终了

**Comment by lpxks** [comment]
哥们 无法记忆大小新版本发布的时候人家说了啊。 再说本来开源就为爱发电，你怎么还抱怨了。反正我作为白嫖怪很感谢这些开源贡献者一直在维护这个项目。

**Comment by itlaohui** [comment]
开源没啥可挑的，虽然2版本有点不好使，但可以等着更新啊，1.7.7挺稳定的就先用1.7.7，等2稳定了就再换回2版本，这是开源项目，觉得哪里不好你可以加入协作者去完善这些漏洞，或者提意见等更新

**Comment by MuFengss** [comment]
首先开源的不止一个，就别拿开源说事了，即使全都收费我也是要用的，其次我说的都是事实，没有夸大抹黑。就拿窗口重置来说，专门修复过一次，结果是怎么修的我一开始也说了，我不知道是反馈的不够清楚没理解，还是压根试都懒得去试呢。本身没有恶意，就是实话实说，如果让你感到不适，抱歉了你就当看个乐子吧！

**Comment by iwalke** [comment]
窗口的问题，维护者说框架的问题，因为升级了框架。等框架修复吧，这个框架bug确实比较多。

2.0.2 相比 1.7.7 的更新，感知比较明显的 1.7.7 每次打开 app 很慢。2 的改善了，只有首次慢，看维护者说是内存常驻了，我内存挺大的，没啥感知，可以接受。当然最好是优化掉，不是常驻，并没有说占内存不重要。。。算是缓解措施吧

mihomo 的 gui 还有 [clash-nyanpasu](https://github.com/LibNyanpasu/clash-nyanpasu)、[FlClash](https://github.com/chen08209/FlClash)、[v2rayN](https://github.com/2dust/v2rayN)，你可以去试试。

如果机场协议是 ss V2Ray Trojan 这些协议，你可以用 [cfw](https://web.archive.org/web/*/https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.39/*)，cfw 虽然停更了，但稳定性我觉得很好。Hysteria 个人认为也并没有很好，mihomo 用这协议占内存，sing-box 不知道，没用过，主要没有 windows 客户端。
